gcsfs/__init__.py

```diff
@@ -1,10 +1,27 @@
+import logging
+import os
+
 from ._version import get_versions
 
+logger = logging.getLogger(__name__)
 __version__ = get_versions()["version"]
 del get_versions
 from .core import GCSFileSystem
 from .mapping import GCSMap
 
+if os.getenv("GCSFS_EXPERIMENTAL_ZB_HNS_SUPPORT", "false").lower() in ("true", "1"):
+    try:
+        from .extended_gcsfs import ExtendedGcsFileSystem as GCSFileSystem
+
+        logger.info(
+            "gcsfs experimental features enabled via GCSFS_EXPERIMENTAL_ZB_HNS_SUPPORT."
+        )
+    except ImportError as e:
+        logger.warning(
+            f"GCSFS_EXPERIMENTAL_ZB_HNS_SUPPORT is set, but failed to import experimental features: {e}"
```

Comment:
What would cause this? google libraries missing or not up to date? Maybe a good idea to give remediation instructions.